src/styles/blocks/content_container.scss

```diff
@@ -0,0 +1,9 @@
+.main {
+  display: flex;
+  min-width: 1024px;
+  max-width: 1200px;
+  justify-content: center;
+  flex-wrap: wrap;
+  padding: 50px 40px;
+  margin: 0 auto;
+}
```

Comment:
It is said in the readme that if you are familiar with grid it is mandatory to use it.
I'm aware of that you have done this task a while ago but I would like you to refactor your layout to grid. And additionally add media queries for [RWD](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Responsive_Design) - remember about mobile first approach.
Here are the [grid layout instrutions](https://github.com/mate-academy/layout_catalog/blob/master/grid.md) from the readme